docs/manuals/user/index.md

```diff
@@ -1,456 +1,324 @@
-
-1. Custom **time index** metadata. The most recent custom time index
-(the value of the `Fiware-TimeIndex-Attribute`)
-attribute value found in any of the attribute metadata sections in the notification.
-See the previous option about the details regarding subscriptions.
-
-1. **TimeInstant** attribute. As specified in the
-[FIWARE IoT agent documentation](https://github.com/telefonicaid/iotagent-node-lib#the-timeinstant-element).
-
-1. **TimeInstant** metadata. The most recent `TimeInstant` attribute value
-found in any of the attribute metadata sections in the notification.
-(Again, refer to the [FIWARE IoT agent documentation](https://github.com/telefonicaid/iotagent-node-lib#the-timeinstant-element).)
-
-1. **timestamp** attribute.
-
-1. **timestamp** metadata. The most recent `timestamp` attribute value found
-in any of the attribute metadata section in the notification.
-As specified in the
-[FIWARE data models documentation](https://fiware-datamodels.readthedocs.io/en/latest/guidelines/index.html#dynamic-attributes).
-
-1. **dateModified** attribute. If you payed attention in the
-[Orion Subscription section](#orion-subscription), this is the `"dateModified"`
-value notified by Orion.
-
-1. **dateModified** metadata. The most recent dateModified attribute value
-found in any of the attribute metadata sections in the notification.
-
-1. Finally, QL will use the **Current Time** (the time of notification
-reception) if none of the above options is present or none of the values found
-can actually be converted to a `datetime`.
-
-### Restrictions and Limitations
-
-- You cannot have two entity types with the same name but different
-capitalisation. E.g: `Preprocessor` and `preProcessor`. The same applies for
-attribute names of a given entity. I.e, attributes `hotSpot` and `hotspot`
-will be treated as the same. These are rare corner-cases, but it is worth
-keeping in mind this. Ultimately, the correct naming of types and attributes
-should respect the naming guidelines explained
-[here](http://fiware-datamodels.readthedocs.io/en/latest/guidelines/index.html).
-
-- Attributes metadata are still not being persisted. See [Issue 12](https://github.com/orchestracities/ngsi-timeseries-api/issues/12)
-
-- While support for multiple data in a single notification as been recently introduced
-  (See [PR 191](https://github.com/orchestracities/ngsi-timeseries-api/pull/191)),
-  The following limitations still apply: a error in a single data entity will invalidate
-  the all set. There is not optimisation for large message size.
-
-- Data are assumed to be consistent. I.e., if the first data notification for
-  an entity type use a given set of data types for the attributes, the following
-  data notifications must be consistent, or they will be rejected. E.g. if
-  the data type of attribute `speed` of entity type `car` is set initially
-  to `Number`, later on it cannot be set to `Text`.
+# QuantumLeap
```

Comment:
remove from the pr the related commit.

Reply:
@chicco785 , This commits is in master branch of my repository so once this [PR](https://github.com/orchestracities/ngsi-timeseries-api/pull/455) and all previous PR will get close it will not reflect in branch commit.

Reply:
but this commit won't be accepted (see my comment on the related one)